compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/constant-propagation-unary.expect.md

```diff
@@ -0,0 +1,86 @@
+
+## Input
+
+```javascript
+function foo() {
+  const a = Math.random();
+  if (!a) {
+    console.log("foo");
+  }
+
+  const b = true;
+  if (!b) {
+    console.log("bar");
+  } else {
+    console.log("baz");
+  }
+
+  console.log({
+    b0: !true,
+    n0: !0,
+    n1: !1,
+    n2: !2,
+    n3: !-1,
+    s0: !"",
+    s1: !"a",
+    s2: !"ab",
+    u: !undefined,
+    n: !null,
+  });
```

Comment:
rather than log this, can you return the value? specifically, try returning `<Stringify value={object} />` (import `Stringify` from "shared-runtime")

Reply:
Updated in c6a277f8616c517433d6bd7d1fade12ab6747049.